cirq-google/cirq_google/experimental/noise_models/calibration_to_fidelity_test.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+import cirq
+import cirq_google
+from cirq.devices.fidelity import Fidelity, NoiseModelFromFidelity
+from cirq_google.api import v2
+from cirq_google.experimental.noise_models.calibration_to_fidelity import fidelity_from_calibration
+from google.protobuf.text_format import Merge
+import numpy as np
+
+def test_fidelity_from_calibration():
+  xeb_1 = 0.999
+  xeb_2 = 0.996
+
+  p00_1 = 0.001
+  p00_2 = 0.002
+  p00_3 = 0.003
+
+  t1_1 = 0.005
+  t1_2 = 0.007
+  t1_3 = 0.003
+
+  _CALIBRATION_DATA = Merge(
+      """
+    timestamp_ms: 1579214873,
+    metrics: [{
+        name: 'xeb',
+        targets: ['0_0', '0_1'],
+        values: [{
+            double_val: """ + str(xeb_1) + """
```

Comment:
You can use [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) for this, e.g.:
```python
f"""
double_val: {xeb_1}
"""
```
Examples of the f-string format are given on the linked page (may need to scroll down a bit)